packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -476,8 +476,20 @@ class Splitter extends (CollectionWidget as any) {
       onResizeStart: (e: ResizeStartEvent): void => {
         const { element, event } = e;
 
-        this._currentLayout = this._layout;
         const $resizeHandle = $(element);
+
+        this._getAction(RESIZE_EVENT.onResizeStart)({
+          event,
+          handleElement: getPublicElement($resizeHandle),
+        });
+
+        // @ts-expect-error typify event members
+        if (event.cancel) {
```

Comment:
`cancel` should be propetry of main arg, not it's `event` field. For example [link](https://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxDraggable/Types/DragStartEvent/#cancel)